src/main/resources/assets/mwc/shaders/reflex.frag

```diff
@@ -40,7 +40,7 @@ void main(){
 
 
 	//gl_FragColor = vec4(0.0, 0.0, 0.2, 0.1) + texture2D(ret, uv + vec2(0.5, 0.5));
-	vec4 retColor = texture2D(ret, uv + vec2(0.5, 0.5));
+	vec4 retColor = texture2D(ret, uv + vec2(0.55, 0.5));
```

Comment:
After thinking about it, it should be in the middle because the deviation is not in the shader but in the models, it will be eventually fixed by having gun specific offset but for now let's keep it in the middle where the shot is fired.